soccer/src/soccer/gameplay/robocup-py.cpp

```diff
@@ -43,6 +44,8 @@ using namespace boost::python;
 #include "planning/MotionConstraints.hpp"
 #include "referee/ExternalReferee.hpp"
 
+#include <rc-fshare/pid.hpp>
```

Comment:
The includes here look a little weird. It might have problems with the using. Also `rj-fshare` doesn't fit the regex, it needs a `-` also in the `([A-Za-z0-9.\/-_])+`, eg `([A-Za-z0-9.\/-_-])+`